tests/regression/results.csv

```diff
@@ -310,30 +322,60 @@ github,                             level 4 with dict dms,              advanced
 github,                             level 4 with dict dds,              advanced one pass,                  41251
 github,                             level 4 with dict copy,             advanced one pass,                  41216
 github,                             level 4 with dict load,             advanced one pass,                  42252
+github,                             level 5 row 1,                      advanced one pass,                  135121
+github,                             level 5 row 1 with dict dms,        advanced one pass,                  38938
+github,                             level 5 row 1 with dict dds,        advanced one pass,                  38741
+github,                             level 5 row 1 with dict copy,       advanced one pass,                  38934
+github,                             level 5 row 1 with dict load,       advanced one pass,                  42252
+github,                             level 5 row 2,                      advanced one pass,                  134584
+github,                             level 5 row 2 with dict dms,        advanced one pass,                  38758
+github,                             level 5 row 2 with dict dds,        advanced one pass,                  38737
+github,                             level 5 row 2 with dict copy,       advanced one pass,                  38759
+github,                             level 5 row 2 with dict load,       advanced one pass,                  42252
```

Comment:
I'm suspicious of this result. Why do we get exactly the same result as with `row = 1` and `row = 2` with `dict load`? The rest are all different.

Reply:
Was actually just looking into this since I noticed this in my separate benchmarking as well initially. It appears to be an existing bug here:
https://github.com/facebook/zstd/blob/980f3bbf8354edec0ad32b4430800f330185de6a/lib/compress/zstd_compress.c#L5170

If the cdict exists but a compression level isn't set, but we reload the dictionary, we end up taking the level of the cdict which is fixed, rather than the `cParams`. I'll put up a separate PR for this since it affects non-row matchfinder results too.

Reply:
You can replicate this  behavior if you hardcode the attach dict pref to forceLoad and run benchmarks like `-b1e19`. Everything is just compressed with default compression level.

Reply:
Yeah, I figured it was something like that, nice find!

I'm fine to fix it in a separate PR, we'll just need to double check the row-based match finder results after that PR, to make sure they make sense.